linux-admin/part-3-tasks/ch-13-shell-history/ch-13-tasks_othman.md

```diff
@@ -0,0 +1,26 @@
+#  ch.13 shell history
+1. list the last 10 commands you wrote
+
```

Comment:
it's more important to mentioned the command you used to find the list not the actual list